react.md

```diff
@@ -27,12 +30,14 @@ Try your best to answer each question on your own before looking up the answer o
  
  //Googled Answer
  
+ In general, a package is simply a folder with code and a package.json file that describes the contents. When you want to use another package, you first need to add it to your dependencies. This means running yarn add [package-name] to install it into your project.
+ Most packages will be installed from the npm registry and referred to by simply their package name. For example, yarn add react will install the react package from the npm registry.
  
 #### 5. There are three mistakes in this code that would cause it to break our application. Find the mistakes and fix them:
 
     import React, { Component } from 'react';
 
-    class Recipes {
+    class Recipes extends Component*(1) {
       constructor(props){
         super(props)
         this.state = {
```

Comment:
In order to use the map function the objects in state would need to be in an array.